1.9/usage/gpu/aws-install.md

```diff
@@ -0,0 +1,39 @@
+---
+post_title: Configure Your AWS Cluster
+nav_title: Configure Your AWS Cloud Cluster
+feature_maturity: experimental
+menu_order: 2
+---
+
+#  Prerequisites
+- An AWS S3 bucket with a [bucket policy](https://dcos.io/docs/1.8/administration/installing/cloud/aws/advanced/aws-custom/) that allows the launched AWS instances to download the files from the S3 bucket.
+- The [AWS CLI](https://aws.amazon.com/cli/) installed.
+- The `zen.sh` script copied to your local machine. The script and instructions are [here](docs/1.9/administration/installing/cloud/aws/advanced/quickstart/).
+
+## Create Dependencies
+
+1. Run the `zen.sh` script to create the Zen template dependencies. These dependencies will be used as input to create your stack in CloudFormation.
+  ```
+  bash ./zen.sh <stack-name>
+  ```
+  **Important:** You must run the `zen.sh` script before performing the next steps.
+
+1. Follow the instructions [here](docs/1.9/administration/installing/cloud/aws/advanced/quickstart/) to create a cluster with advanced AWS templates, but use the following GPU-specific configuration.
```

Comment:
Global: relative links must have the forward slash`/`, as in `(/docs/1.9/administration/installing/cloud/)`